workspace/r/ds-pack/v1/Dockerfile

```diff
@@ -0,0 +1,93 @@
+FROM rocker/verse:4.2.0
+
+# Adding dependencies for top 100 CRAN Packages
+RUN apt-get update \
+    # installation requirements
+    && apt-get install -y --no-install-recommends \
+    apt-transport-https \
+    curl \
+    gnupg2 \
+    software-properties-common \
+    lsb-release \
+    # docker repo
+    && curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg \
+    && echo \
+        "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu \
+        $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+    build-essential \
+    docker-ce \
+    gdal-bin \
+    git \
+    libcairo2-dev \
+    libcurl4-openssl-dev \
+    libfontconfig1-dev \
+    libfreetype6-dev \
+    libfribidi-dev \
+    libgdal-dev \
+    libgeos-dev \
+    libharfbuzz-dev \
+    libicu-dev \
+    libjpeg-dev \
+    libpng-dev \
+    libproj-dev \
+    libssh2-1-dev \
+    libssl-dev \
+    libtiff-dev \
+    libudunits2-dev \
+    libxml2-dev \
+    make \
+    pandoc \
+    zlib1g-dev \
+    # Platform Essential Packages
+    curl \
+    jq \
+    wget \
+    # Add extras for helpfulness
+    fish \
+    htop \
+    nano \
+    nano \
+    vim \
+    zsh \
+    # For AWS CLI help
+    groff \
+    # ODBC Driver Manager (for snowflake)
+    unixodbc \
+    unixodbc-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install Python + all required packages
+COPY install_packages.sh /tmp/install_packages.sh
+RUN bash /tmp/install_packages.sh
+
```

Comment:
@saurabh0402  should we add 
`ARG PEAK_USER_ID
ENV PEAK_USER_ID=$PEAK_USER_ID`

Reply:
We don't need non-root user ID for workspaces as in workspace user any way has non-root access only.

Reply:
hey @saurabh0402 in that case can we update the security description in the ticket

Reply:
Ahh, thanks. Updated.